### ISSUE TITLE
Linter v2 does not show errors in GoLand

### ISSUE THREAD

**Issue by PerryProjects** [golangci/golangci-lint#5679]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/usage/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

go install

### Description of the problem

Can it be that the Linter v2 is not marking the errors correctly in GoLand, or is there a trick to get the errors underlined again? I'm using WSL on Windows. The errors appear correctly in the console. In v1, you had to set up a file watcher, but there is no mention of that under integrations anymore.

### Version of golangci-lint

<details>

```console
$ golangci-lint --version
golangci-lint has version v2.0.2 built with go1.24.1 from (unknown, modified: ?, mod sum: "h1:dMCC8ikPiLDvHMFy3+XypSAuGDBOLzwWqqamer+bWsY=") on (unknown)
```

</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
go version go1.24.1 linux/amd64
AR='ar'
CC='gcc'
CGO_CFLAGS='-O2 -g'
CGO_CPPFLAGS=''
CGO_CXXFLAGS='-O2 -g'
CGO_ENABLED='1'
CGO_FFLAGS='-O2 -g'
CGO_LDFLAGS='-O2 -g'
CXX='g++'
GCCGO='gccgo'
GO111MODULE='on'
GOAMD64='v1'
GOARCH='amd64'
GOAUTH='netrc'
GOBIN=''
GOCACHE='/home/user/.cache/go-build'
GOCACHEPROG=''
GODEBUG=''
GOENV='/home/user/.config/go/env'
GOEXE=''
GOEXPERIMENT=''
GOFIPS140='off'
GOFLAGS=''
GOGCCFLAGS='-fPIC -m64 -pthread -Wl,--no-gc-sections -fmessage-length=0 -ffile-prefix-map=/tmp/go-build2835698506=/tmp/go-build -gno-record-gcc-switches'
GOHOSTARCH='amd64'
GOHOSTOS='linux'
GOINSECURE=''
GOMOD='/var/www/project/go.mod'
GOMODCACHE='/home/user/go/pkg/mod'
GONOPROXY=''
GONOSUMDB=''
GOOS='linux'
GOPATH='/home/user/go'
GOPRIVATE=''
GOPROXY='https://proxy.golang.org,direct'
GOROOT='/usr/local/go'
GOSUMDB='sum.golang.org'
GOTELEMETRY='local'
GOTELEMETRYDIR='/home/user/.config/go/telemetry'
GOTMPDIR=''
GOTOOLCHAIN='auto'
GOTOOLDIR='/usr/local/go/pkg/tool/linux_amd64'
GOVCS=''
GOVERSION='go1.24.1'
GOWORK=''
PKG_CONFIG='pkg-config'

```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
INFO golangci-lint has version v2.0.2 built with go1.24.1 from (unknown, modified: ?, mod sum: "h1:dMCC8ikPiLDvHMFy3+XypSAuGDBOLzwWqqamer+bWsY=") on (unknown)
INFO [config_reader] Config search paths: [./ /var/www/project /var/www /var / /home/user]
INFO [config_reader] Used config file .golangci.yml
INFO maxprocs: Leaving GOMAXPROCS=20: CPU quota undefined
INFO [goenv] Read go env for 60.752199ms: map[string]string{"GOCACHE":"/home/user/.cache/go-build", "GOROOT":"/usr/local/go"}
INFO [lintersdb] Active 70 linters: [asasalint asciicheck bidichk bodyclose canonicalheader copyloopvar cyclop depguard dupl durationcheck errcheck errname errorlint exhaustive exptostd fatcontext forbidigo funlen gocheckcompilerdirectives gochecksumtype gocognit goconst gocritic gocyclo godot goimports golin
es gomoddirectives goprintffuncname gosec govet iface ineffassign intrange loggercheck makezero mirror mnd musttag nakedret nestif nilerr nilnesserr nilnil noctx nolintlint nonamedreturns nosprintfhostport perfsprint predeclared promlinter protogetter recvcheck revive rowserrcheck sloglint spancheck sqlclosecheck staticcheck testableexamples testifylint testpackage tparallel unconvert unparam unused usestdlibvars usetesting wastedassign whitespace]
INFO [loader] Go packages loading at mode 8767 (types_sizes|files|imports|name|compiled_files|deps|exports_file) took 1.06044673s
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 1.344882ms
INFO [linters_context/goanalysis] analyzers took 19.35355767s with top 10 stages: buildir: 8.044669727s, the_only_name: 3.511339978s, goimports: 2.021361018s, exhaustive: 1.68790443s, musttag: 837.237072ms, inspect: 390.623408ms, fact_deprecated: 370.581377ms, printf: 291.678784ms, ctrlflow: 233.043476ms, wastedassign: 184.525543ms
INFO [runner/exclusion_rules] Skipped 66 issues by rules: [Text: "exported \\S+ \\S+ should have comment( \\(or a comment on this block\\))? or be unexported", Linters: "revive"]
WARN [runner/exclusion_rules] Skipped 0 issues by rules: [Text: "package comment should be of the form \".+\"", Source: "// ?(nolint|TODO)", Linters: "revive"]
WARN [runner/exclusion_rules] Skipped 0 issues by rules: [Text: "comment on exported \\S+ \\S+ should be of the form \".+\"", Source: "// ?(nolint|TODO)", Linters: "revive, staticcheck"]
INFO [runner/exclusion_rules] Skipped 2 issues by rules: [Path: "_test\\.go", Linters: "bodyclose, dupl, errcheck, funlen, goconst, gosec, noctx, wrapcheck"]
WARN [runner/exclusion_rules] Skipped 0 issues by rules: [Source: "TODO", Linters: "godot"]
INFO [runner/exclusion_rules] Skipped 10 issues by rules: [Text: "should have a package comment", Linters: "revive"]
INFO [runner/exclusion_rules] Skipped 3 issues by rules: [Text: "shadow: declaration of (\"err\") shadows declaration", Linters: "govet"]
INFO [runner] Issues before processing: 119, after processing: 7
INFO [runner] Processors filtering stat (in/out): invalid_issue: 119/119, exclusion_rules: 119/37, max_from_linter: 7/7, path_shortener: 7/7, severity-rules: 7/7, path_absoluter: 119/119, cgo: 119/119, exclusion_paths: 119/119, diff: 9/9, uniq_by_line: 9/7, max_same_issues: 7/7, source_code: 7/7, path_prettifier: 7/7, path_relativity: 119/119, nolint_filter: 37/9, fixer: 9/9, sort_results: 7/7, filename_unadjuster: 119/119, generated_file_filter: 119/119, max_per_file_from_linter: 7/7
INFO [runner] processing took 906.523µs with stages: nolint_filter: 494.908µs, exclusion_rules: 202.012µs, generated_file_filter: 134.675µs, path_relativity: 26.689µs, source_code: 19.831µs, invalid_issue: 4.85µs, cgo: 4.1µs, sort_results: 3.838µs, path_absoluter: 3.16µs, filename_unadjuster: 2.903µs, uniq_by_line: 2.35µs, max_same_issues: 2.127µs, path_shortener: 1.534µs, max_from_linter: 1.041µs, max_per_file_from_linter: 1.025µs, path_prettifier: 616ns, fixer: 365ns, diff: 228ns, exclusion_paths: 207ns, severity-rules: 64ns
INFO [runner] linters took 3.517697312s with stages: goanalysis_metalinter: 3.516728596s
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
Hello,

If you are using the Go Linter plugin: https://github.com/golangci/golangci-lint/issues/3870

I'm using Goland (but not the plugin) and everything works.
